docs/examples/widgets/progress_bar_gradient.py

```diff
@@ -0,0 +1,34 @@
+from textual.app import App, ComposeResult
+from textual.color import Gradient
+from textual.containers import Center, Middle
+from textual.widgets import ProgressBar
+
+
+class ProgressApp(App[None]):
+    """Progress bar with a rainbow gradient."""
+
+    def compose(self) -> ComposeResult:
+        gradient = Gradient.from_colors(
+            "#881177",
+            "#aa3355",
+            "#cc6666",
+            "#ee9944",
+            "#eedd00",
+            "#99dd55",
+            "#44dd88",
+            "#22ccbb",
+            "#00bbcc",
+            "#0099cc",
+            "#3366bb",
+            "#663399",
+        )
+        with Center():
+            with Middle():
```

Comment:
Bit of an aside, but I often find myself defining a `CenterMiddle`.

Reply:
Not a bad idea.